kombu/transport/SQS.py

```diff
@@ -160,15 +152,23 @@ def _reset_cycle(self):
 
     def entity_name(self, name, table=CHARS_REPLACE_TABLE):
         """Format AMQP queue name into a legal SQS queue name."""
-        return text_t(safe_str(name)).translate(table)
```

Comment:
@jseutter did that not me. 
.fifo suffix means it's a FIFO queue, i.e. retains ordering. this is a new feature in SQS. non-FIFO queues don't preserve ordering.